tensornetwork/network_components.py

```diff
@@ -1075,9 +1074,9 @@ def disconnect(self,
     if self.is_dangling():
       raise ValueError("Cannot break dangling edge {}.".format(self))
     if not edge1_name:
-      edge1_name = '__unnamed_edge__'
+      edge1_name = '__broke_edge1__{}'.format(self.name)
```

Comment:
Not a fan of "break," "broke," or "broken" for describing edge that have been disconnected. Either "disconnected" or "cut" would be preferable. I would go with "disconnected" to avoid confusion. Docstring should also be changed to avoid using "break."

Reply:
Done, replaced `broken` with `disconnected`